geppetto.js/geppetto-ui/src/3d-canvas/threeDEngine/MeshFactory.js

```diff
@@ -26,6 +26,27 @@ export default class MeshFactory {
     this.linesThreshold = linesThreshold;
     this.particleTexture = particleTexture;
     this.THREE = THREE ? THREE : require('three');
+
+    this.setupLoaders()
+  }
+
+  setupLoaders (){
+    const dracoLoader = new DRACOLoader()
+    dracoLoader.setDecoderPath('https://raw.githubusercontent.com/mrdoob/three.js/dev/examples/js/libs/draco/gltf/');
```

Comment:
before we had also 'https://raw.githubusercontent.com/mrdoob/three.js/dev/examples/js/libs/draco/' ,
does the gltf decode path work for "pure" draco and gltf+draco?